docs/modules/reference/pages/manage/app/detail.adoc

```diff
@@ -1,124 +0,0 @@
-= Navigating the Application Detail Page
```

Comment:
This move broke the navigation link mentioned in     "Application Management.adoc"